direct-auth-samples/src/main/resources/templates/enroll-authenticators.html

```diff
@@ -55,7 +55,7 @@ <h2 class="text-center">Enroll Authenticator</h2>
         </div>
         <tr th:each="item : ${factorList}">
             <div class="form-check">
-                <input class="form-check-input" type="radio" name="authenticator-type" th:value="${item}">
+                <input class="form-check-input" type="radio" name="authenticator-type" th:value="${item}" checked="true">
```

Comment:
This is to prevent the error when user clicks `Proceed` when no radio button is selected.